Qubino flush 1 relay and dimmer fixes

bendeitch opened the pull request:
Adds the Momentary Button Child Device to the Qubino Flush Dimmer and Qubino Flush 1 Relay.
Fix issue with Hubitat when changing child device types.
Changes the Z-Wave commands used for on/off/setLevel/refresh and poll to improve response time for Qubino Flush Dimmer.

bendeitch committed:
Add Momentary Button Child Device

bendeitch committed:
Remove Qubino Flush Dimmer metadata elements not used in Hubitat

bendeitch committed:
Use switchMultiLevel for Qubino Flush Dimmer to improve response time and support dimming duration

bendeitch committed:
Adjust Qubino Flush Dimmer to delete and re-add child devices

Hubitat does not allow the device type of a child device to be changed once created. This change also removes child devices when they are 'Disabled'.

bendeitch committed:
Add Momentary Button Child Device to Qubino Flush Dimmer

bendeitch committed:
Remove Qubino Flush 1 Relay metadata elements not used in Hubitat

bendeitch committed:
Adjust Qubino Flush 1 Relay to delete/re-add when changing child device type

bendeitch committed:
Add Momentary Button Child Device to Qubino Flush 1 Relay

bendeitch committed:
Fix issue where null duration caused dimming failure

bendeitch committed:
Reorder sendEvent statement to avoid race condition

erocm123 reviewed (changes requested):
Sorry for the delay at getting to this. I like the momentary button option. 

There are a couple things that I have questions about. On some of the z-wave command calls you are formatting the command and then the "command" method is formatting it again. I thought this might cause problems, but it is working for you?

erocm123 committed:
Slight modification for sending commands

The "command" method accepts a delay parameter so I'm changing the delayBetween calls. Also removing the .format() from some z-wave commands as they are getting created with .format().format()

erocm123 commented:
I made the changes and will test it over the next couple days. Thanks for the contributions!

erocm123 committed:
Removing "response" from updated()

erocm123 committed:
Remove "response" from updated() method

erocm123 commented:
I think it look good. Thanks for the updates!
